modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -180,8 +195,8 @@ public PartitionReplicaLifecycleManager(
             TopologyService topologyService,
             LowWatermark lowWatermark,
             ExecutorService ioExecutor,
-            ScheduledExecutorService rebalanceScheduler
-    ) {
+            ScheduledExecutorService rebalanceScheduler,
+            ClockService clockService, PlacementDriver placementDriver) {
```

Comment:
One at a row, please. Please also move `) { `to a separate row.

Reply:
Done